ucan/Cargo.toml

```diff
@@ -47,6 +48,9 @@ getrandom = { version = "~0.2", features = ["js"] }
 tokio = { version = "^1", features = ["macros", "test-util"] }
 
 [dev-dependencies]
-did-key = "0.1"
-serde_ipld_dagcbor = "~0.2"
-wasm-bindgen-test = "~0.3"
+# NOTE: Using updated fork, due to issues w/ bitvec/funty 1.2.0 yank and
+# signature_bls crate:
+# Related: <https://github.com/decentralized-identity/did-key.rs/issues/31>
+did-key = { git = "https://github.com/Satellite-im/did-key.rs", branch = "backport-patch" }
```

Comment:
`did-key` has made a release 7d ago so the git ref can be removed in favor of 0.2